src/index.ts

```diff
@@ -113,7 +119,7 @@ async function main(): Promise<string[] | undefined> {
     });
   }
 
-  if (build == null) {
+  if (build === null) {
```

Comment:
Don't do this

`== null` catches `undefined` and `null`
```suggestion
  if (build == null) {
```

Reply:
Fine to go this way but...

IMO it'd be better to be explicit that you're checking both instead of relying on the implicit knowledge that `==` checks both. Also will build ever be undefined? We type it as `Build | null` so in theory it won't be.

Either way, could we assume build isn't a boolean/string/number and do

```ts
if(!build) { ... }
```

Reply:
Overall, I'm moving away from really ever using "null". Having said that...

One problem here is the type system doesn't initialize anything. So that `Build | null` should be initialized to `null` explicitly, otherwise it's `undefined`.

I agree, it would be an easier rule to always say "use `===`". That's why I've gone toward just never using null!

`(!build)` works for me.